pywhat/Data/regex.json

```diff
@@ -1547,5 +1547,18 @@
       "Tags": [
          "Credentials"
       ]
+   },
+   {
+      "Name": "Nano (NANO) Wallet Address",
+      "Regex": "^(nano|xrb)_[13]{1}[13456789abcdefghijkmnopqrstuwxyz]{59}$",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 0.8,
```

Comment:
This is probably 1 because it require having the string `nano` or `xrb` in it :)

Reply:
i thought so too but I didn't want to over estimate. I can make that change today. What about the 10 checks failing?

Reply:
Try running `pytest` on your machine inside of the PyWhat directory, so that tests run and you can see the reason why ones are failing.

And don't forget to run `black` and `isort` for your code formatting. :)

Reply:
will do :)